test/modules/toolbox/components/exportMfpToolContent/ExportMfpToolContent.test.js

```diff
@@ -386,14 +416,36 @@ describe('ExportMfpToolContent', () => {
 
 			const toggleButton = element.shadowRoot.querySelector('#autorotation');
 
-			toggleButton.click();
+			toggleButton.dispatchEvent(getToggleEventFor(toggleButton));
```

Comment:
What's the cause of this workaround?

Reply:
The 'toggleButton' was a Checkbox component before, which have a click-action.
Now the 'toggleButton' is a Toggle component, which does not register a EventListener for 'click'-events.